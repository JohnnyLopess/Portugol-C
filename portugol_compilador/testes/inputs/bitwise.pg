programa{
    funcao inicio() {
        escreva(5 & 3)
        escreva(~7)
        escreva(5 | 3)
        escreva(23 << 1)
        escreva(105 >> 1)
        escreva(5 ^ 3)
    }
}