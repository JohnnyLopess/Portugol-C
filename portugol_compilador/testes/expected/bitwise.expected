#include <stdio.h>

int main() {
    printf("%d\n", 5 & 3);
    printf("%d\n", ~7);
    printf("%d\n", 5 | 3);
    printf("%d\n", 23 << 1);
    printf("%d\n", 105 >> 1);
    printf("%d\n", 5 ^ 3);
    return 0;
}
